Scripts/Enemy/Enemy.cs

```diff
@@ -23,16 +24,23 @@ public override void _Ready()
 
         MeshInstance cylinder = GetNode<MeshInstance>("MeshInstance");
         cylinder.SetSurfaceMaterial(0, material);
+
+        player = GetTree().Root.GetNode<Spatial>("Level").GetNode<Player>("Player");
+
+        RegenerationRate = 0.0f;
     }
 
-    public override void _PhysicsProcess(float delta)
+    public override void _Process(float delta)

```

Comment:
How come change from physics process to process?

Reply:
The UI stuff I think should put in Process, only physics-related stuff should put in PhysicsProcess I think. But so far all of our stuff we put in PhysicsProcess, which I'm not sure whether it's optimal or not.

Reply:
Oh I thought is use either one or the other

Reply:
Ok but now its a bit funny cos the function is Process and it calls Base.PhysicsProcess

Reply:
got meh, i tot it's removed

Reply:
Gg see wrongly